### PR TITLE
chore: semantic release

### PR DESCRIPTION
**Pull request by tsirlucas**
Removes lerna anduses semantic-release directly in order to have more customization

### CODE REVIEW

**Comment by tsirlucas** [comment]
forced push for better organizing commits

File: package/bin/before-tag.sh [hunk 1]
```diff
@@ -2,7 +2,7 @@
 # shellcheck disable=SC2103
 
 
-# Runs before lerna tagging in order to set correct tags for internal native-package
+# Runs before semantic-relase tagging in order to set correct tags for internal native-package
```
Comment (vishalnarkhede) [inline package/bin/before-tag.sh:5]:
```suggestion
# Runs before semantic-release tagging in order to set correct tags for internal native-package
```

File: release/release.config.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    ],
+  ];
+
+  if ((process.env.GH_TOKEN || process.env.GITHUB_TOKEN) && process.env.GIT_BRANCH === 'master') {
+    plugins.push([
+      '@semantic-release/git',
+      {
+        assets: [
+          `${process.cwd()}/package.json`,
+          `${process.cwd()}/yarn.lock.json`,
```
Comment (vishalnarkhede) [inline release/release.config.js:62]: .json?

File: release/next.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+configPromise.then((config) => {
+  return semanticRelease({
+    ...config,
+    branches: [
+      'master',
+      { name: 'develop', channel: 'next', prerelease: `next.${GITHUB_SHORT_SHA}` },
+    ],
+  }).then((result) => {
+    // Maybe this breaks releases because without a tag semantic-release cant keep track of whats already deployed
+    // talk to vish
```
Comment (vishalnarkhede) [inline release/next.js:15]: ??

**Pull request merged**